JetBrains/compose-multiplatform
# Update gradle plugin: Deprecate experimental configuration for Compose for Web

Since 1.6.10 Compose for Web goes to Alpha and experimental configuration is not needed anymore. We'll configure the web targets by default when they're added to projects.


## Testing
- I built the plugin to mavenLocal. And used it in a couple of our samples.
- After gradle sync completes, I observe the Deprecation warning in build.gradle.kts on `compose.experimental.web` usages (see a screenshot below)
<img width="1022" alt="Screenshot 2024-05-10 at 15 41 14" src="https://github.com/JetBrains/compose-multiplatform/assets/7372778/e8ede073-8d34-4dd7-ae74-c83ca0ff5c96">


Then I remove deprecated API usages and test that the project works without it:
- run `./gradlew clean` just in case
- run the app: `./gradlew wasmJsBrowserRun` and `./gradlew jsBrowserRun` - both run fine
- build the production distribution: `./gradlew wasmJsBrowserDistribution`
- `cd ..../build/dist/wasmJs/productionExecutable`  and run `python -m http.server`, open a browser at `http://localhost:8000` - the app should work the same way it works with gradle tasks above


This should be tested by QA

## Release Notes

### Highlights - Web
- Compose for Web goes to Alpha!
Some experimental Compose Multiplatform Gradle plugin APIs for web app configuration were deprecated. Their usage is not required anymore.
